app/views/shared/eligible_region_content_components/_professional_recognition.html.erb

```diff
@@ -80,13 +58,45 @@
   <% end %>
 <% end %>
 
-<% if region.sanction_check_none? %>
-  <p class="govuk-body">
-    You also need to show evidence of your work history.
-  </p>
+<% if region.sanction_check_written?%>
+  <p class="govuk-body"><%= t("eligibility_interface.three_months_before_applying") %></p>
+<% end %>
+
+<%= render "shared/teaching_authority_contact_information", region: %>
+
+<% if region.status_check_online? || region.sanction_check_online? %>
   <p class="govuk-body">
-    For each school this should include:
+    As your education department or authority has an online register of teachers,
+    you’ll also need to provide any reference numbers we need to check your record.
   </p>
+<% end %>
+
+<% if region.status_check_online? %>
+  <% if region.teaching_authority_status_information.present?%>
+    <%= raw GovukMarkdown.render(region.teaching_authority_status_information) %>
+  <% end %>
+
+  <% if region.country.teaching_authority_status_information.present?%>
+    <%= raw GovukMarkdown.render(region.country.teaching_authority_status_information) %>
+  <% end %>
+<% end %>
+
+<% if region.sanction_check_online? %>
+  <% if region.teaching_authority_sanction_information.present?%>
+   <%= raw GovukMarkdown.render(region.teaching_authority_sanction_information) %>
+  <% end %>
+
+  <% if region.country.teaching_authority_sanction_information.present?%>
+    <%= raw GovukMarkdown.render(region.country.teaching_authority_sanction_information) %>
+  <% end %>
+<% end %>
+
+
+<% if region.teaching_authority_other.present? %>
+  <%= raw GovukMarkdown.render(region.teaching_authority_other) %>
+<% end %>
```

Comment:
I think this should go below the work history section, next to the one for `region.country`.